tests/unit/components/Scorecard.spec.ts

```diff
@@ -0,0 +1,117 @@
+    const wrapper = mount(Scorecard);
+    await wrapper.find('[data-test="share"]').trigger("click");
+
+    expect(mockWindowOpen).toHaveBeenCalled();
+  });
+
+  it("share text on whatsapp when no questions answered", async () => {
+    const wrapper = mount(Scorecard);
+    await wrapper.find('[data-test="share"]').trigger("click");
+
+    expect(mockWindowOpen).toHaveBeenCalledWith(
+      "https://api.whatsapp.com/send/?phone&text=%F0%9F%8E%89%F0%9F%8E%8A%F0%9F%8E%89%F0%9F%8E%8A%F0%9F%8E%89%F0%9F%8E%8A%F0%9F%8E%89%F0%9F%8E%8A%F0%9F%8E%89%F0%9F%8E%8A%0A%0A%F0%9F%8F%86%20*Hooray!%20I%20completed%20a%20Quiz!*%20%F0%9F%8F%86%0A%0A%F0%9F%8C%9F%20*undefined*%20%F0%9F%8C%9F%0A%0AI%20answered%20undefined%20questions%20with%200%25%20accuracy%20on%20Avanti%20Fellows%20quiz%20today!%20%F0%9F%98%87%0A%0A%F0%9F%8E%89%F0%9F%8E%8A%F0%9F%8E%89%F0%9F%8E%8A%F0%9F%8E%89%F0%9F%8E%8A%F0%9F%8E%89%F0%9F%8E%8A%F0%9F%8E%89%F0%9F%8E%8A"
+    );
+  });
+
+  it("share text on whatsapp when one question answered", async () => {
+    const progressPercentage = 50;
+    const wrapper = mount(Scorecard);
+    await wrapper.setProps({
+      numQuestionsAnswered: 4,
+      progressPercentage: progressPercentage,
+    });
+    await flushPromises();
+    await jest.advanceTimersByTime(1000);
+
+    await wrapper.find('[data-test="share"]').trigger("click");
+
+    expect(mockWindowOpen).toHaveBeenCalledWith(
+      `https://api.whatsapp.com/send/?phone&text=%F0%9F%8E%89%F0%9F%8E%8A%F0%9F%8E%89%F0%9F%8E%8A%F0%9F%8E%89%F0%9F%8E%8A%F0%9F%8E%89%F0%9F%8E%8A%F0%9F%8E%89%F0%9F%8E%8A%0A%0A%F0%9F%8F%86%20*Hooray!%20I%20completed%20a%20Quiz!*%20%F0%9F%8F%86%0A%0A%F0%9F%8C%9F%20*undefined*%20%F0%9F%8C%9F%0A%0AI%20answered%204%20questions%20with%200%25%20accuracy%20on%20Avanti%20Fellows%20quiz%20today!%20%F0%9F%98%87%0A%0A%F0%9F%8E%89%F0%9F%8E%8A%F0%9F%8E%89%F0%9F%8E%8A%F0%9F%8E%89%F0%9F%8E%8A%F0%9F%8E%89%F0%9F%8E%8A%F0%9F%8E%89%F0%9F%8E%8A`
+    );
+  });
+
+  it("share text on whatsapp when multiple questions answered", async () => {
+    const progressPercentage = 50;
+    const wrapper = mount(Scorecard);
+    await wrapper.setProps({
+      numQuestionsAnswered: 4,
+      progressPercentage: progressPercentage,
+    });
+    await flushPromises();
+    await jest.advanceTimersByTime(1000);
+
+    await wrapper.find('[data-test="share"]').trigger("click");
+
+    expect(mockWindowOpen).toHaveBeenCalledWith(
+      `https://api.whatsapp.com/send/?phone&text=%F0%9F%8E%89%F0%9F%8E%8A%F0%9F%8E%89%F0%9F%8E%8A%F0%9F%8E%89%F0%9F%8E%8A%F0%9F%8E%89%F0%9F%8E%8A%F0%9F%8E%89%F0%9F%8E%8A%0A%0A%F0%9F%8F%86%20*Hooray!%20I%20completed%20a%20Quiz!*%20%F0%9F%8F%86%0A%0A%F0%9F%8C%9F%20*undefined*%20%F0%9F%8C%9F%0A%0AI%20answered%204%20questions%20with%200%25%20accuracy%20on%20Avanti%20Fellows%20quiz%20today!%20%F0%9F%98%87%0A%0A%F0%9F%8E%89%F0%9F%8E%8A%F0%9F%8E%89%F0%9F%8E%8A%F0%9F%8E%89%F0%9F%8E%8A%F0%9F%8E%89%F0%9F%8E%8A%F0%9F%8E%89%F0%9F%8E%8A`
+    );
+  });
+});
```

Comment:
why is this test not included from Plio?
<img width="615" alt="Screenshot 2022-04-08 at 2 54 49 PM" src="https://user-images.githubusercontent.com/13185699/162407160-4f28f0df-6628-4bd2-915e-be21a766fcee.png">

Reply:
I removed it because initially there was no isshown in the file, later on after our discussion, I added isshown but did not add this test, will add it now.